packages/scripts/config/webpack.config.js

```diff
@@ -321,7 +343,7 @@ const scriptConfig = {
 				cleanStaleWebpackAssets: false,
 			} ),
 
-		new RenderPathsPlugin(),
+		new PhpFilePathsPlugin( { props: [ 'render', 'variations' ] } ),
```

Comment:
Do we expect to have something apart from `props` in the future? I'm asking because I was wondering if we should just pass an array with the properties, but maybe having the object with `props` is clear. I don't have an opinion, just sharing random thoughts 😄

Reply:
Yeah, it's kind of an ugly signature 😅 I tried to stick with the format that I've seen in [Webpack's docs on writing a plugin](https://webpack.js.org/contribute/writing-a-plugin/), where the plugin constructor accepts an `options` object (i.e. with named properties).

On a more pragmatic note, I actually have a WIP PR (#63311) that would another argument, so it might be a good idea to stick to this format (even if it's a bit ugly 😉).

Reply:
Great! That makes sense to me 🙂